outputs.tf

```diff
@@ -8,6 +8,11 @@ output "dynamodb_table_id" {
   value       = element(concat(aws_dynamodb_table.this.*.id, [""]), 0)
 }
 
+output "dynamodb_table_name" {
```

Comment:
There is no `name` in attributes, but `id`. https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dynamodb_table#id

Reply:
correct - but we are allowed to reference any of the input attributes which allows this to work
<img width="388" alt="image" src="https://user-images.githubusercontent.com/10913471/121022985-76e24a80-c768-11eb-89b0-01e7cd8c3760.png">

not sure if its needed, but I can see how users could be confused between `id` and `name` - it was added just due to #27 but am happy to remove it not warranted

Reply:
We need to use only attributes (`id`) to make sure that Terraform dependency graph is respected and resource actually creates before its attributes can be returned. It can be a problem when referencing an argument (eg, `name`) but the resource is not **yet** created, and Terraform fails.

Similar behavior as when outputting static `var.name`.

Hope it makes sense. Please remove.

Reply:
done